regex-syntax/src/parser.rs

```diff
@@ -1047,19 +1058,19 @@ fn rev_concat(mut exprs: Vec<Expr>) -> Expr {
     }
 }
 
-// Returns ture iff the given character is allowed in a capture name.
+// Returns true if the given character is allowed in a capture name.
```

Comment:
Yeah, I can't recall it showing up much in papers. It's more like "I'm hand writing a proof and I'm going to gratuitously abuse shorthand."